src/Vts/MonteCarlo/Helpers/LayerTissueRegionToolbox.cs

```diff
@@ -12,18 +13,50 @@ public abstract class LayerTissueRegionToolbox
         /// This extends photon ray from position and angle of exit from tissue into air
         /// up to plane specified by zPlane
         /// </summary>
-        /// <param name="p">position of photon exit</param>
-        /// <param name="d">direction of photon exit</param>
+        /// <param name="photonPos">position of photon exit</param>
+        /// <param name="photonDir">direction of photon exit</param>
         /// <param name="zPlane">z-plane above tissue in air, must be negative value</param>
         /// <returns>position on plane extended to</returns>
-        public static Position RayExtendToInfinitePlane(Position p, Direction d, double zPlane)
+        public static Position RayExtendToInfiniteZPlane(Position photonPos, Direction photonDir, double zPlane)
```

Comment:
Changing the name of a public method is a breaking change. Usually we would create another method with the new name and then mark the old one as obsolete.

Reply:
"RayExtendToInfinitePlane" was used by one detector named "SurfaceFibereDetector". As RayExtendToInfinitePlane was ambiguous, it was changed to RayExtendToInfiniteZPlane.

Reply:
@janakarana since this is a public method it is available outside of the library also so changing a name is a breaking change. To change the name, you can rename the method with the new name, add back in the method with the old name, mark it as obsolete and then just call the new one passing the parameters. If you need assistance with this, let us know.

Reply:
An example of the Obsolete attribute:
[Obsolete("This method was replaced by x, please use the new method")]

Reply:
Revert "RayExtendToInfiniteZPlane" to original "RayExtendToInfinitePlane"